common/h/dyn_regs.h

```diff
@@ -60,6 +60,7 @@ namespace Dyninst
         Arch_amdgpu_vega  =  0x84000000,
         Arch_cuda         =  0x88000000,
         Arch_amdgpu_cdna2 =  0x94000000, //future support for cdna2
+        Arch_amdgpu_gfx908 =  0x98000000, //future support for gfx908
```

Comment:
Is gfx908 the "official" AMD name? I ask because it varies substantially from the previous ones.

Reply:
Yes.
In later patches vega and cdna2 will be renamed to similar style.

Reply:
We should let the Rice folks know about this soon. If they've made a release of HPCToolkit that uses the vega/cdna2 names from Dyninst, that will complicate how we handle this.